### DISCUSSION TITLE
Egress from a static IP

### DISCUSSION THREAD

**Discussion by tuomassiren** [aws/copilot-cli#4326] [General]
There's already a discussion https://github.com/aws/copilot-cli/discussions/1914 regarding outbound internet traffic and a comment:
"Hi @inssein following up on this, now with copilot v1.5.0 we do support creating NAT Gateways! https://github.com/aws/copilot-cli/releases/tag/v1.5.0 🚀"

Is there a way to specify the elastic IP address in manifest?
If not, is the creation of an external VPC the only way?

**Comment by paragbhingre** [comment]
@tuomassiren, We cannot use existing elastic IP addresses for your NAT gateways as Copilot creates new elastic IP addresses. But I think you will have to import your external VPC with NAT gateways created using existing elastic IP addresses.

**Reply by tuomassiren** [reply]
Assuming I have VPC with a NAT GW with private subnets routed to the NAT GW, would this work?
For some reason I'm seeing public subnets internet gw as the egress inteface. 

network:
  vpc:
    placement: private
    id: vpc-0f27f1bd600a30b23
    subnets:
      public:
        - id: subnet-0a325b6d619525f62
        - id: subnet-012acc46ea5811b02
      private:
        - id: subnet-0196afdf8f0bab3fe
        - id: subnet-0df06de8c5465f899

**Comment by tuomassiren** [comment]
Found the problem:
I had this snippet in the environment manifest and not in the service manifest.
network:
    placement: 'private'

**Reply by paragbhingre** [reply]
I am glad you could spot the problem! Please reach out to us if you have any more questions.
